<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1311</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>1291</width>
      <height>651</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>G.employee</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>G.clients</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>G.services</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>G.fournisseurs</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>G.produits</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>60</y>
        <width>601</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="rechercher">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>20</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Rechercher par ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idrech">
      <property name="geometry">
       <rect>
        <x>552</x>
        <y>30</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Trier">
      <property name="geometry">
       <rect>
        <x>1170</x>
        <y>60</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Trier</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="impc">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>230</y>
        <width>611</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Impdos_2">
      <property name="geometry">
       <rect>
        <x>1180</x>
        <y>270</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Imprimer</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>501</width>
        <height>451</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="stackedWidgetPage1">
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>431</width>
          <height>391</height>
         </rect>
        </property>
        <property name="title">
         <string>Ajouter Produit</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>90</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Libelle</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>130</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Prix Achat</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="prix">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>120</y>
           <width>151</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>170</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Date Achat</string>
         </property>
        </widget>
        <widget class="QDateEdit" name="date">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>160</y>
           <width>151</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>200</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Quantit√©</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Id </string>
         </property>
        </widget>
        <widget class="QLineEdit" name="qte">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>200</y>
           <width>151</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="id">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>40</y>
           <width>151</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_ajouter">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>340</y>
           <width>111</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>240</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>classification</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="classification">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>240</y>
           <width>151</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>340</y>
           <width>111</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Annuler</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>280</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>id fournisseur </string>
         </property>
        </widget>
        <widget class="QLineEdit" name="idf">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>280</y>
           <width>151</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="libelle">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>80</y>
           <width>151</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QCommandLinkButton" name="commandLinkButton">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>340</y>
           <width>61</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="stackedWidgetPage2">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>431</width>
          <height>411</height>
         </rect>
        </property>
        <property name="title">
         <string>Modifier Produit </string>
        </property>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Id</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>130</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Prix Achat </string>
         </property>
        </widget>
        <widget class="QLineEdit" name="ident">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>40</y>
           <width>151</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="priix">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>120</y>
           <width>151</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_modifier">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>340</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Modifier</string>
         </property>
        </widget>
        <widget class="QPushButton" name="supprimer">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>340</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>170</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Date Achat</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="cls">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>240</y>
           <width>151</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="qte2">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>200</y>
           <width>151</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>210</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Quantite</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>240</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>classification</string>
         </property>
        </widget>
        <widget class="QDateEdit" name="date2">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>160</y>
           <width>151</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>340</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Annuler</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>280</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>id fourniasseur</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="idf2">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>280</y>
           <width>151</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Libelle</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lib">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>80</y>
           <width>151</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QCommandLinkButton" name="commandLinkButton_2">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>340</y>
           <width>61</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="ok">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>40</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>ok</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="image_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>480</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>image</string>
      </property>
     </widget>
     <widget class="QLabel" name="image1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>460</y>
        <width>161</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>460</y>
        <width>231</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="augmenter">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>500</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>augmenter</string>
      </property>
     </widget>
     <widget class="QLabel" name="Digtal_clock">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>430</y>
        <width>211</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>clock</string>
      </property>
     </widget>
     <widget class="QLabel" name="DateTime">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>520</y>
        <width>261</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1311</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
